Quadriga/src/main/java/edu/asu/spring/quadriga/web/transformation/ShowTranformationFilesUiController.java

```diff
@@ -0,0 +1,24 @@
+package edu.asu.spring.quadriga.web.transformation;
+
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+/**
+ * @Controller
+ * @author JayaVenkat
+ *
+ */
+@Controller
+public class ShowTranformationFilesUiController {
+	
+	/**
+	 * 
+	 * This method will show uploadTranformation.jsp to user 
+	 * @return
+	 */
+	@RequestMapping( value="auth/transformation/selectTransformationFiles",method=RequestMethod.GET)	
+	public String selectTransformationFiles(HttpServletRequest request){				
```

Comment:
why do you inject a request object here? 

Also, is this the method that shows the list of transformations? 

path should have project id in it like the other pages.
Project owner, project admins, and collaborators should have access to this, so you need to use an access policy.